macros/tests.sql

```diff
@@ -150,15 +188,26 @@
   {% set actual_query = test_queries.actual_query %}
   {% set expectations_query = test_queries.expectations_query %}
   {% set test_query = test_queries.test_query %}
+  {% set udf_definitions = test_configuration.options.udf_definitions | default("udf_definitions") %}
```

Comment:
For some reason when testing this, the default value was a plain text string `udf_definitions` and caused the query to fail on BigQuery. Replaced by :
```
{% set udf_definitions = test_configuration.options.udf_definitions | default("") %}
```
and worked fine!

Reply:
Same for the other macro where you define `udf_definitions`. If that worked fine for you maybe I don't have something locally that you do, but still!